cypress/integration/statistic/statistic.test.js

```diff
@@ -0,0 +1,134 @@
+/// <reference types="cypress" />
+import { config } from '../../../src/configs';
+
+import {
+  getPopularCategories,
+  getPopularProducts,
+  getPopularCategoriesEmpty,
+  getPopularProductsEmpty,
+  getPaidOrdersStatistic,
+  getOrdersStatistic
+} from './statistic.variables';
+
+describe('Statistic page test', () => {
+  beforeEach(() => {
+    cy.login(Cypress.env('ADMIN_LOGIN'), Cypress.env('ADMIN_PASSWORD'));
+    cy.stubRequest('getPopularCategories', getPopularCategories).as(
+      'getPopularCategories'
+    );
+    cy.stubRequest('getPopularProducts', getPopularProducts).as(
+      'getPopularProducts'
+    );
+    cy.visit('/');
+    cy.wait(3000);
+
+    cy.get('[href="/stats"]').click();
+  });
+
+  it('should be visible and have default settings', () => {
+    cy.get('[data-cy=page-title]').should('be.visible');
+    cy.get('[data-cy=page-title]').contains(
+      config.titles.statisticTitles.mainPageTitle
+    );
+
+    cy.get('[data-cy=date-selector]').should('be.visible');
+    cy.get('[data-cy=date-selector]').click();
+    cy.get('[data-cy="date-select-За 7 Днів"]').should('be.visible');
+    cy.get('[data-cy="date-select-За 14 Днів"]').should('be.visible');
+    cy.get('[data-cy="date-select-За місяць"]').should('be.visible');
+    cy.get('[data-cy="date-select-За 3 місяці"]').should('be.visible');
+    cy.get('[data-cy="date-select-За рік"]').should('be.visible');
+    cy.get('.MuiMenu-paper').invoke('hide');
+    cy.get('[data-cy=date-selector]').contains(
+      config.labels.doughnut.dateMenuOptions[0].label
+    );
+
+    cy.get('[data-cy=bar-selector]').should('be.visible');
+    cy.get('[data-cy=bar-selector]').contains(
+      config.labels.bar.select[0].label
+    );
+
+    cy.get('[data-cy=doughnut-selector]').should('be.visible');
+    cy.get('[data-cy=doughnut-selector]').contains(
+      config.labels.doughnut.select[0].label
+    );
+
+    cy.get('[data-cy=bar-chart-box]').should('be.visible');
+    cy.get('[data-cy=bar-chart-box]').children('canvas').should('be.visible');
+    cy.get('[data-cy=total-count]').should('be.visible');
+    cy.get('[data-cy=total-count]').contains('120');
```

Comment:
maybe it will be good to move this value to variable

Reply:
solved